tutorials/general/t8_tutorial_search.cxx

```diff
@@ -190,27 +202,28 @@ t8_tutorial_search_query_callback (t8_forest_t forest, t8_locidx_t ltreeid, cons
   T8_ASSERT (particles_per_element != NULL);
   T8_ASSERT (query != NULL);
 
-  /* Test whether this particle is inside this element. */
-  particle_is_inside_element
-    = t8_forest_element_point_inside (forest, ltreeid, element, particle->coordinates, tolerance);
-  if (particle_is_inside_element) {
-    if (is_leaf) {
-      /* The particle is inside and this element is a leaf element.
+  /* Test whether the particles are inside this element. */
+  t8_forest_element_point_batch_inside (forest, ltreeid, element, coords, num_active_queries, query_matches, tolerance);
```

Comment:
Note, this function got renamed in the other PR, right? Might cause a merge conflict one way or another.

Reply:
Yes, but as currently none of them is merged I will wait for the first one to merge and adapt the other one accordingly.